import sys


def safe_print_integer_err(value):
    try:
        print("{:d}".format(value))
    except (ValueError, TypeError):
        print("Exception: {}".format(e), file=sys.stderr)
        return False
    return True
